### PR TITLE
Remove qf section from success view

### PR DESCRIPTION
**Pull request by MohammadPCh**
- #4042

### CODE REVIEW

File: src/components/views/donate/DonateIndex.tsx [hunk 1]
```diff
@@ -40,7 +40,7 @@ import { DonatePageProjectDescription } from './DonatePageProjectDescription';
 const DonateIndex: FC = () => {
 	const { formatMessage } = useIntl();
 	const { isMobile } = useDetectDevice();
-	const { project, isSuccessDonation, hasActiveQFRound } = useDonateData();
+	const { project, successDonation, hasActiveQFRound } = useDonateData();
```
Comment (jainkrati) [inline src/components/views/donate/DonateIndex.tsx:43]: why was this renamed?

Comment (MohammadPCh) [reply src/components/views/donate/DonateIndex.tsx:43]: @jainkrati, it was renamed because the previous name, 'isSuccessDonation,' was incorrect as it suggested a boolean type, but it actually includes the success donation data.

**Review by jainkrati — APPROVED** [review]
